### PR TITLE
Gallery: fixed text edit page layout

### PR DESCRIPTION
**Pull request by FloVanGH**

### CODE REVIEW

File: examples/gallery/ui/pages/text_edit_page.slint [hunk 1]
```diff
@@ -10,6 +10,7 @@ export component TextEditPage inherits Page {
     description: @tr("Similar to LineEdit, but can be used to enter several lines of text. The widget can be imported from \"std-widgets.slint\".");
 
     HorizontalBox {
+        vertical-stretch: 1;
         GroupBox {
             vertical-stretch: 0;
```
Comment (ogoffart) [inline examples/gallery/ui/pages/text_edit_page.slint:15]: I believe the vertical-stretch here and in the other GroupBox are counter-productive, are they not?

Comment (FloVanGH) [reply examples/gallery/ui/pages/text_edit_page.slint:15]: yeah the `vertical-stretch: 0;` is not needed. Maybe an old copy paste fragment.

Comment (FloVanGH) [reply examples/gallery/ui/pages/text_edit_page.slint]: I have removed it from the groupbox

**Comment by tronical** [comment]
Fixed in bbefdbe4df1a30a54d463752dae27c84eccab1c7

**Pull request closed without merging**